Stonx/Stonx/Controller/Settings/PersonalnfoViewController.swift

```diff
@@ -4,108 +4,89 @@
 //
 //  Created by Angel Zambrano on 10/31/22.
 //
-
+import Parse
 import UIKit
-// TODO: present errors. For example when information is incorrect. 
+
 class PersonalnfoViewController: UIViewController {
     let tableview = UITableView(frame: .zero, style: .grouped)
     let settings = ["username", "full name"]
-    
+    let currentUser = PFUser.current()!
     
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .white
-        // Do any additional setup after loading the view.
         
         viewSetups()
         addconstraints()
     }
-    
-    private func viewSetups(){
+        
+    private func viewSetups() {
         tableview.translatesAutoresizingMaskIntoConstraints = false
         tableview.register(BalanceTableViewCell.self, forCellReuseIdentifier: BalanceTableViewCell.identifier)
         tableview.dataSource = self
         tableview.delegate = self
         view.addSubview(tableview)
-        
     }
     
     private func addconstraints() {
         tableview.anchor(top: view.safeAreaLayoutGuide.topAnchor, leading: view.leadingAnchor, bottom: view.safeAreaLayoutGuide.bottomAnchor, trailing: view.trailingAnchor)
-        
     }
-    
-
-    /*
-    // MARK: - Navigation
-
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
-        // Get the new view controller using segue.destination.
-        // Pass the selected object to the new view controller.
-    }
-    */
-
 }
 
 extension PersonalnfoViewController: UITableViewDataSource {
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         return 2
     }
     
+    // Username and email
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell = tableView.dequeueReusableCell(withIdentifier: BalanceTableViewCell.identifier, for: indexPath) as! BalanceTableViewCell
-        // configuring the title
         if indexPath.row == 0 {
             cell.accessoryType = .disclosureIndicator
-            cell.configure(with: "username")
-            cell.configure(name: "angelzzz23")
+            cell.configure(with: "Username")
+            cell.configure(name: currentUser.username!)
             return cell
         } else {
             cell.accessoryType = .disclosureIndicator
-            cell.configure(with: "Full Name")
-            cell.configure(name: "Angel Zambrano")
+            cell.configure(with: "Email")
+            cell.configure(name: currentUser.email!)
```

Comment:
this will throw an error if the user's email field is empty. Might want to provide an empty string instead of force unwrapping